package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "portal-cms",
-    "version": "0.0.0",
+    "version": "2.1.0",
     "license": "MIT",
     "description": "The core CMS codebase for all new and updated TACC CMS sites.",
```

Comment:
This is not quite the CMS codebase, but rather some client side script bundles, right?

Reply:
Correct.

Because of the dependencies, this is like the version of the "Core CMS Stylesheet Build Process".